apps/synapse-portal-framework/src/components/ampals/AMPALSResearchPageLayout.tsx

```diff
@@ -0,0 +1,105 @@
+import { Box, Container, ContainerProps, Typography } from '@mui/material'
+import { PropsWithChildren, ReactNode } from 'react'
+
+type AMPALSResearchPageLayoutProps = PropsWithChildren<{
+  ContainerProps?: ContainerProps
+  headerTitle?: string
+  headerImageURL?: string
+  sidebarTitle?: string
+  sidebarContent?: ReactNode
+}>
+
+const AMPALSResearchPageLayout = (props: AMPALSResearchPageLayoutProps) => {
+  const {
+    ContainerProps,
+    children,
+    headerTitle,
+    headerImageURL,
+    sidebarTitle,
+    sidebarContent,
+  } = props
+  return (
+    <Container
+      {...ContainerProps}
+      disableGutters={true}
+      maxWidth={false}
+      sx={{
+        overflow: 'auto',
+        ...ContainerProps?.sx,
+      }}
+    >
+      <Box
+        sx={{
+          width: '100%',
+          height: '180px',
+          backgroundColor: '#0655B0',
+          backgroundImage: `url(${headerImageURL})`,
+          backgroundSize: 'cover',
+          backgroundRepeat: 'no-repeat',
+          display: 'flex',
+          alignItems: 'center',
+        }}
+      >
+        <Typography
+          variant="h2"
+          sx={{
+            marginLeft: '80px',
+            color: '#FFFFFF',
+            fontWeight: 700,
+            fontSize: '48px',
+          }}
+        >
+          {headerTitle}
+        </Typography>
+      </Box>
+      <Box
+        sx={{
+          display: 'flex',
+          flexDirection: 'row',
+          gap: '80px',
+          padding: '60px 80px',
+        }}
+      >
+        <Box
+          sx={{
+            display: 'flex',
+            flexDirection: 'column',
+            width: '360px',
+            gap: '20px',
+          }}
+        >
+          <Typography
+            sx={{
+              fontWeight: 700,
+              fontSize: '24px',
+              color: '#353A3F',
+            }}
+          >
+            {sidebarTitle}
+          </Typography>
+          {sidebarContent}
+        </Box>
+        <Box
+          sx={{
+            marginTop: '0px',
+            width: '720px',
```

Comment:
Not sure how this would look with narrow screen sizes with this hard-coded width. What's the purpose of this? If just to constrain the max width, then I suggest

```suggestion
            maxWidth: '720px',
```

but it might be appropriate instead to use a MUI Container here -- not totally sure about that though

Reply:
What makes you think that a container might be more appropriate?

Reply:
By my eye the content is not centered horizontally. So I'm going to move forward with `maxWidth`.

![Screenshot 2025-05-02 at 9 17 15 AM](https://github.com/user-attachments/assets/92c4f30a-422a-4432-9d62-345f929f73e6)